### ISSUE TITLE
Bug: Incorrectly added text as language to code block

### ISSUE THREAD

**Issue by asknet** [rvben/rumdl#257]
Hi,

For the following text, linter reported language missing and fixed on it's own by adding text in the closing code block marker.

Before:
```
    - Sample code:
    - ```java
          List<Map<String,String>> inputs = new List<Map<String,String>>();
      ```
```

After:
```
    - Sample code:
    - ```java
          List<Map<String,String>> inputs = new List<Map<String,String>>();
      ```text
```

Thank you!

**Comment by rvben** [comment]
Thanks @asknet for raising the issue! Fixed in v0.0.209 - refactored to use pulldown-cmark for proper code block detection.
